samples/ScreenRecordingSample/Platforms/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true"></application>
+	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true" android:requestLegacyExternalStorage="true" android:preserveLegacyExternalStorage="true">
+		<service android:name="Plugin.Maui.ScreenRecording.ScreenRecordingImplementation.ScreenRecordingService" android:exported="false" android:foregroundServiceType="mediaProjection" />
+	</application>
+	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+	<uses-permission android:name="android.permission.RECORD_AUDIO" />
+	<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
Looks like for API 34 (or even earlier?) we need the `FOREGROUND_SERVICE_MEDIA_PROJECTION` permission as well

Reply:
I'm an apple guy lol, my dev android only goes to 31, but that is good to know!  :)  It works on there.

Reply:
`MediaProjection` is null in the Android implementation on line 164

Reply:
I ran it before I pushed.  I will verify when I get home.

Reply:
Yeah not sure what is going on for API 34, but seems we need an extra check there maybe